packages/apidom-ns-workflows-1/src/refractor/specification.ts

```diff
@@ -61,6 +61,8 @@ const specification = {
               $ref: '#/visitors/document/objects/Info',
             },
             sourceDescriptions: SourceDescriptionsVisitor,
+            workflows: WorkflowVisitor,
```

Comment:
```suggestion
            workflows: WorkflowsWorkflowsVisitor,
```
NOTE: This visitor is not implemented yet along with it's NCE